swift/Tests/Blockchains/MultiversXTests.swift

```diff
@@ -53,12 +53,12 @@ class ElrondTests: XCTestCase {
     }
     
     func testSignGenericActionUndelegate() {
-        // Successfully broadcasted https://explorer.elrond.com/transactions/3301ae5a6a77f0ab9ceb5125258f12539a113b0c6787de76a5c5867f2c515d65
+        // Successfully broadcasted https://explorer.MultiversX.com/transactions/3301ae5a6a77f0ab9ceb5125258f12539a113b0c6787de76a5c5867f2c515d65
```

Comment:
shall we use lowercases? https://explorer.multiversx.com...

Reply:
Fixed.